### PR TITLE
Update UI

### LINKED ISSUE #128: Upgrade UI

### PR DESCRIPTION
**Pull request by Stefan-Jiang**
Change the theme colour Fixed #128

### CODE REVIEW

**Review by Stefan-Jiang — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/ui/BookCardWithCover.java [hunk 1]
```diff
@@ -64,15 +64,15 @@ public BookCardWithCover(Book book, int displayedIndex) {
         this.book = book;
         id.setText(displayedIndex + ". ");
         name.setText(book.getName().fullName);
-        isbn.setText("isbn: " + book.getIsbn().value);
-        address.setText(book.getAddress().value);
-        email.setText("contact: " + book.getEmail().value);
-        times.setText("Number of time: " + book.getTimes().value);
+        isbn.setText("ISBN              " + book.getIsbn().value);
```
Comment (HCY123902) [inline src/main/java/seedu/address/ui/BookCardWithCover.java:67]: Thanks for changing the letter. It seems to be more consistent.

File: src/main/resources/view/Base.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
+//@@author Stefan-Jiang-reused
+//Adapted from tP code of CS2103T-T10-4
+
+.background {
```
Comment (HCY123902) [inline src/main/resources/view/Base.css:4]: Thanks for adding the colour theme and changing the user interface.

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -35,7 +35,7 @@
 
         <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
           <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
+            <Insets top="10" right="10" bottom="10" left="10" />
```
Comment (HCY123902) [inline src/main/resources/view/MainWindow.fxml:38]: Thanks for adjusting the layout.

**Review by HCY123902 — COMMENTED** [review]
LGTM

**Pull request merged**